### PR TITLE
Feature: add firewall config

### PR DESCRIPTION
**Pull request by wrnu**
Adds configuration of the firewall and the perimeter alb target groups

### CODE REVIEW

File: modules/firewall/variables.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+variable "alb_fqdn" {
+
```
Comment (sheaphillips) [inline modules/firewall/variables.tf]: could we include type and description for the variables pls?

**Review by sheaphillips — CHANGES_REQUESTED** [review]
Minor twiddle requested...

And a general question/food for thought..how much does this change if/when we switch firewalls?

File: modules/firewall/variables.tf (hunk 1 above)
Comment (wrnu) [reply modules/firewall/variables.tf]: Done and pushed

Comment (wrnu) [reply modules/firewall/variables.tf]: > Minor twiddle requested...
> 
> And a general question/food for thought..how much does this change if/when we switch firewalls?

It really depends on how they work, if they use the same virtual ip concept it should be fairly similar. We'd just create a new firewall module

**Pull request merged**